package ExceptionHandeling;

public class Throw {
	// Auto-generated method stub
	private static void divideByZero() {
		
    // throw an exception
    throw new ArithmeticException("Trying to divide by 0");
	}
	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
		divideByZero();
	
	}


}

